dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/search/AdvancedSearchPage.java

```diff
@@ -238,9 +238,13 @@ private boolean isDateField(DatasetFieldType datasetFieldType) {
                                                   BundleUtil.getStringFromBundle("description"),
                                                   BundleUtil.getStringFromBundle("advanced.search.files.description.tip")));
 
-        filesSearchFields.add(new TextSearchField(SearchFields.FILE_TYPE_SEARCHABLE,
-                                                  BundleUtil.getStringFromBundle("advanced.search.files.fileType"),
-                                                  BundleUtil.getStringFromBundle("advanced.search.files.fileType.tip")));
+//        filesSearchFields.add(new TextSearchField(SearchFields.FILE_TYPE_SEARCHABLE,
+//                                                  BundleUtil.getStringFromBundle("advanced.search.files.fileType"),
+//                                                  BundleUtil.getStringFromBundle("advanced.search.files.fileType.tip")));
```

Comment:
Do we need this commented out?

Reply:
I wanted to keep it because we're probably gonna add a task to add this field to advanced search again, just in a bit different form and this code will work for it. 
However I agree that its bad to leave comments like that so I'll remove it.